internal/pkg/cli/app_show.go

```diff
@@ -139,23 +146,35 @@ func (opts *ShowAppOpts) retrieveData() error {
 	}
 
 	var serializedEnvs []serializedLoadBalancedWebAppEnv
+	var serializedServices []serializedLoadBalancedWebAppService
 	for _, env := range environments {
 		webAppURI, err := opts.identifier.URI(env.Name)
-		if err == nil {
-			serializedEnvs = append(serializedEnvs, serializedLoadBalancedWebAppEnv{
-				Name:   env.Name,
-				Region: env.Region,
-				Prod:   env.Prod,
-				URL:    webAppURI.DNSName,
-				Path:   webAppURI.Path,
-			})
+		if err != nil && !applicationNotDeployed(err) {
```

Comment:
Should we consider changing `applicationNotDeployed` to `applicationIsDeployed` reversing its return value to prevent this double negative case? It current reads like "not application not deployed" and I'd prefer it read "application is deployed".